scripts/generate-artifacts.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
overall this file could probably use a bunch more verbose logging for each step that it's taking.

Reply:
now that we have this mechanism - could you also find all instances of comments like the following, and remove them from the codebase?

https://github.com/proofcarryingdata/zupass/blob/c85a67df7cea2b54e4bb732e72be43657903f8f0/packages/ethereum-ownership-pcd/src/EthereumOwnershipPCD.ts#L27-L32

Reply:
That package is not included in the packages supported by the CLI because there're no circuits there. It is using Semaphore right? Do you want `pcd-artifacts` to handle them too?

Reply:
I'm not sure about the best way forward regarding this question, I'll defer to you.

Reply:
I think those artifacts too shouldn't be saved on Github, but they could need a different solution. I can open another issue for them and work on that separately.